apps/cli-daemon/src/app/generators/utils.ts

```diff
@@ -10,29 +10,25 @@ const formatSchemaName = (name: string): string => {
     .map((name) => capitalize(name))
     .join('');
 };
-export const getGeneratorDefinition = (
-  generatorName: string,
+export const formatJsonToJs = <T>(path: string): T =>
+  JSON.parse(readFileSync(path, 'utf-8'));
+export const getGeneratorsDefinition = (
```

Comment:
Consider adding try-catch here as well (not required if formatJsonToJs is already wrapped)